### ISSUE TITLE
Warning message about Unknown keyword

### ISSUE THREAD

**Issue by ppatierno** [strimzi/strimzi-kafka-bridge#234]
Investigate and avoid the following warning on startup:

```
2019-06-05 16:48:23,768 [     vert.x-eventloop-thread-1] JsonMetaSchema                 WARN  Unknown keyword example - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
```

**Comment by ShubhamRwt** [comment]
@scholzj @ppatierno Hi, I guess this issue can be closed now, I tested this with bridge 0.21.4 and when I deploy the bridge, I am not getting this `unknown keyword` warning now

**Comment by ppatierno** [comment]
Thanks for reporting @ShubhamRwt it means that they fixed somehow during the Vert.x Web update the OpenAPI side.
